src/livecodes/languages/jsx/jsx-runtime.ts

```diff
@@ -0,0 +1,11 @@
+export const reactRuntime = `
+import React from "react";
```

Comment:
Though, this is a bit different from my expectation of "jsx-runtime".

https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
I should have said "JSX Transform".